mappings/net/minecraft/entity/player/PlayerInventory.mapping

```diff
@@ -31,6 +31,7 @@ CLASS arn net/minecraft/entity/player/PlayerInventory
 	METHOD g getHotbarSize ()I
 	METHOD g setCursorStack (Laxi;)V
 	METHOD h getEmptySlot ()I
+	METHOD h hasItemStack (Laxi;)Z
```

Comment:
containsItemStack would fit better with Java parlance

Reply:
`containsStack` potentially to match with `setCursorStack`